external/mmsegmentation/segmentation_tasks/sample/sample.py

```diff
@@ -0,0 +1,163 @@
+# Copyright (C) 2021 Intel Corporation
```

Comment:
the same question here https://github.com/openvinotoolkit/training_extensions/pull/972#discussion_r828151909

@evgeny-izutov ?